### ISSUE TITLE
NTR: Modifier: MRI: U fiber involvement in subcortical white matter abnormality (Lab: radiology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8938]
New term request Modifier: MRI: U fiber involvement in subcortical white matter abnormality (forsyt_101119171301):   UMLS: 5232624

**Comment by pnrobinson** [comment]
@MickeySegal

this is a bundled term. Can we specify the abnormality of the U fibers?

**Comment by MickeySegal** [comment]
How about "U fiber reduction seen on imaging".  https://www.thieme-connect.com/products/ejournals/pdf/10.1055/a-1207-1006.pdf "Assessment of U-fiber involvement in addition to symmetry is necessary for a closer differential-diagnostic classification."

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000995
- label: Subcortical U fiber lesion
- parents: [Abnormal cerebral subcortex morphology (HP:0010993)](https://hpo.jax.org/app/browse/term/HP:0010993)
- pmid: PMID:38223048
- def: Abnormal morphology of subcortical U fibers, typically as assessed by abnromal intensity on magnetic resonance imaging of the brain.
- comment: Subcortical U-fibers are special types of short association fibers located in the superficial white matter (SWM). In contrast to the SWM, the U-fibers only connect the adjacent gyri. Since U-fibers have a rich blood supply from long perforating arteries and short perforating arteries that span the white matter (WM) and adjacent cortex, they are usually unaffected in chronic ischemic WM disease. Anatomically, U-fibers originate from pyramidal neurons of layer 6 of the cerebral cortex and provide local integration of different sites of the same gyrus or immediately adjacent gyri. Developmentally, U-fibers do not fork until the second half of gestation when gyration and sulcation of the cerebral cortex occur. As a result, U-fibers are not readily involved in diseases of myelin metabolism, such as adrenoleukodystrophy and metachromatic leukodystrophy. However, in demyelinating diseases with normal myelin and oligodendrocyte dominance, including multiple sclerosis and progressive multifocal leukoencephalopathy, U-fibers are commonly involved Pathologic processes that involve the white matter typically appear hyperintense on T2/FLAIR images.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8938
- synonyms: Subcortical U fiber involvement(NONE)
